src/config/ast.rs

```diff
@@ -34,6 +33,22 @@ impl Spec {
         }
     }
 }
+impl From<String> for Spec {
```

Comment:
Implementing from `&str` would reduce repeated code as it avoids the repetition of `to_owned()` function call.